WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersFlow.swift

```diff
@@ -1,19 +1,25 @@
-import UIKit
+import Foundation
 
-extension BlogDetailsViewController {
-    static func presentBloggingRemindersSettingsFlow(from viewController: UIViewController, for blog: Blog, source: BloggingRemindersTracker.FlowStartSource) {
+class BloggingRemindersFlow {
+    static func present(from viewController: UIViewController, for blog: Blog, source: BloggingRemindersTracker.FlowStartSource) {
 
         let blogType: BloggingRemindersTracker.BlogType = blog.isHostedAtWPcom ? .wpcom : .selfHosted
 
         let tracker = BloggingRemindersTracker(blogType: blogType)
         tracker.flowStarted(source: source)
 
         // TODO: Check whether we've already presented this flow to the user. @frosty
-        let flowIntroViewController = BloggingRemindersFlowIntroViewController(tracker: tracker)
+        let flowIntroViewController = BloggingRemindersFlowIntroViewController(blogURIRepresentation: blog.objectID.uriRepresentation(), tracker: tracker)
```

Comment:
Why can't we just pass a blog in here, and the view controller take the uriRepresentation itself?

Reply:
Since the VCs are keeping strong references to this, I wanted to avoid passing a full CoreData object.  But I'm changing this to be a blog, as it may make the logic easier to comprehend.